www/src/routes/MobileLanding.tsx

```diff
@@ -38,17 +53,27 @@ const MobileLanding: FC = () => {
                             </Typography>
                         </Grid>
                     </Grid>
-                    <Grid container item alignItems='center' justify='center' direction='column' spacing={2}>
-                        <Grid container item alignItems='center' justify='center'>
-                            <Paper component='form' className={classes.text_input_root}>
-                                <InputBase className={classes.text_input} placeholder='Email Address' />
-                            </Paper>
-                        </Grid>
-                        <Grid container item alignItems='center' justify='center'>
-                            <Button variant='contained' className={classes.main_button}>
-                                Submit
-                            </Button>
-                        </Grid>
+                    <Grid container item justify='center' alignItems='center'>
+                        <MailchimpSubscribe
+                            url={config.mailchimpUrl}
+                            render={({ subscribe, status }) => (
+                                <Grid container item justify='center' alignItems='center'>
+                                    <Form
+                                        textPlaceholder='Email Address'
+                                        callback={(email) => {
+                                            subscribe({ EMAIL: email });
+                                        }}
+                                        buttonPlaceholder='Submit'
+                                        valueValidator={(email) => validator.isEmail(email)}
+                                    />
+                                    <Grid item>
+                                        <Typography align='center' variant='body1' style={{ fontWeight: 200 }}>
+                                            {getMessage(status)}
+                                        </Typography>
+                                    </Grid>
+                                </Grid>
+                            )}
+                        />
```

Comment:
Is this identical to the one for the browser page? Could this just be it's own component?

Reply:
Yeah good point, can do a quick refactor